app/sass/pages/_api.scss

```diff
@@ -102,27 +107,14 @@
 }
 
 
-
```

Comment:
Why remove this?

Reply:
probably not removed with intent and was a result of copying/pasting/editing other CSS

I'm a big fan of deterministic formatting (e.g. Prettier) so that these issues never come up